### PR TITLE
Update cookie value

### PR DESCRIPTION
**Pull request by situplastik**
PR for #118

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -123,49 +123,62 @@ import CookieConsent, { Cookies, getCookieConsentValue } from "react-cookie-cons
...
+| disableButtonStyles      |                   boolean                   | false                                | If enabled the button will have no default style. (you can still supply style through props)                                            |
+| enableDeclineButton      |                   boolean                   | false                                | If enabled the decline button will be rendered                                                                                          |
+| flipButtons              |                   boolean                   | false                                | If enabled the accept and decline buttons will be flipped                                                                               |
+| ButtonComponent          |               React component               | button                               | React Component to render as a button.                                                                                                  |
```
Comment (Mastermindzh) [inline README.md:175]: The code looks good but this change is a bit overkill imo :)

File: src/index.js [hunk 2]
```diff
@@ -412,6 +436,7 @@ CookieConsent.defaultProps = {
   hideOnAccept: true,
   hideOnDecline: true,
   location: OPTIONS.BOTTOM,
+  visible: VISIBLE_OPTIONS.BY_COOKIE_VALUE,
```
Comment (Mastermindzh) [inline src/index.js:439]: yes, good!

File: README.md (hunk 1 above)
Comment (situplastik) [reply README.md:175]: yep, I tried to follow the same structure but as the second column is long, edit all table was necessary :)

**Comment by Mastermindzh** [comment]
I will merge this soon. Am currently not feeling well but saw this email and wanted to give an update.
Expect it merged by the end of the week.

**Pull request merged**